family2family/src/main/resources/static/js/admin.jsx

```diff
@@ -0,0 +1,218 @@
+class User extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {id: props.user.id, firstName: props.user.firstName, lastName: props.user.lastName,
+            admin: props.user.admin, active: props.user.active, showReply: {false}};
+    }
+
+    onClick(e){
+        e.preventDefault();
+        this.setState({showReply: !this.state.showReply})
+    }
+
+    displayEdit(e){
+        e.preventDefault();
+        console.log("Edit Collapse");
+        if(document.getElementsByClassName("edit-collapse")[0].style.display == 'block')
+        {
+            document.getElementsByClassName("edit-collapse")[0].style.display = 'none';
+        }
+        else
+        {
+            document.getElementsByClassName("edit-collapse")[0].style.display = 'block';
+        };
+    }
+
+    render() {
+        return(
+            <div>
+                <a className="row" data-toggle="collapse" onClick={this.displayEdit.bind(this)}>
+                    <div className="col-sm-3">{this.state.id}</div>
+                    <div className="col-sm-3">{this.state.firstName + " " + this.state.lastName}</div>
+                    <div className="col-sm-3">
+                        <input type="checkbox" name="admin" checked={this.state.admin} disabled={true}/>
+                    </div>
+                    <div className="col-sm-3">
+                        <input type="checkbox" name="active" checked={this.state.active} disabled={true}/>
+                    </div>
+                </a>
+               <div className="edit-collapse">
+                   <EditUserRow id={this.state.id} firstName={this.state.firstName}
+                    lastName={this.state.lastName} admin={this.state.admin} active={this.state.active}/>
+               </div>
+            </div>
+        )
+    }
+}
+class UserTable extends React.Component {
```

Comment:
Nitpicking, but can we have better formatting here? Add a line break before the new class and indent the constructor.